enhancements/installer/vsphere-ipi-zonal.md

```diff
@@ -398,30 +600,39 @@ type Platform struct {
 	// +optional
 	DefaultMachinePlatform *MachinePool `json:"defaultMachinePlatform,omitempty"`
 	// Network specifies the name of the network to be used by the cluster.
-	Network string `json:"network,omitempty"`
+	// Deprecated: Use FailureDomains.Topology.Network
+	DeprecatedNetwork string `json:"network,omitempty"`
 	// DiskType is the name of the disk provisioning type,
 	// valid values are thin, thick, and eagerZeroedThick. When not
 	// specified, it will be set according to the default storage policy
 	// of vsphere.
 	DiskType DiskType `json:"diskType,omitempty"`
-	// vcenters holds the connection details for services to communicate with vCenter.
+	// VCenters holds the connection details for services to communicate with vCenter.
 	// Currently only a single vCenter is supported.
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:validation:MaxItems=1
+	// +kubebuilder:validation:MaxItems=3
 	// +kubebuilder:validation:MinItems=1
 	VCenters []VCenter `json:"vcenters,omitempty"`
-	// failureDomains holds the VSpherePlatformFailureDomainSpec which contains
+	// FailureDomains holds the VSpherePlatformFailureDomainSpec which contains
 	// the definition of region, zone and the vCenter topology.
 	// If this is omitted failure domains (regions and zones) will not be used.
 	// +kubebuilder:validation:Optional
 	FailureDomains []FailureDomain `json:"failureDomains,omitempty"`
+
+	// LoadBalancer defines how the load balancer used by the cluster is configured.
+	// LoadBalancer is available in TechPreview.
+	// +optional
+	LoadBalancer *configv1.VSpherePlatformLoadBalancer `json:"loadBalancer,omitempty"`
+	// Hosts defines network configurations to be applied by the installer. Hosts is available in TechPreview.
+	Hosts []*Host `json:"hosts,omitempty"`
```

Comment:
Is this intended to support hosts as zones? What is the use case I wonder.

Reply:
Pretty sure the answer is yes, afaik it for static ip.

@vr4manta @rvanderp3 do you want to take that one?

Reply:
But this is not reflected in `Infrastructure` type above. So information will be lost after cluster installation?

Reply:
Also - I wonder why is this information here and not in `Topology` type..

Reply:
oh I think I misunderstood, `Hosts` is for OCP hosts and static ip, that isn't ESXi hosts.